Apply.html

```diff
@@ -1,39 +1,44 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
+<!DOCTYPE html>
+<html lang="en">
 <head>
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<title>Apply a job</title>
-<link rel="stylesheet" type="text/css" href="tryonce.css" /> 
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Apply a job</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"> 
+     <link rel="stylesheet" type="text/css" href="tryonce.css" />
 </head>
 
 <body>
-<table class="tableform">
-<form name="apply_form" action="apply.html" method="post">
-    <tr>
-        <th><b>Apply</b></th>
-    </tr>
-    <tr>
-        <td>Username</td>
-        <td><input type="text" name="username" size="25" maxlength="25" class="textbox"></td>
-    </tr>
-    <tr>
-        <td>Job</td>
-        <td><input type="text" name="occupation" size="25" maxlength="25" class="textbox"></td>	
-    </tr>
-    <tr>
-        <td><br /></td>
-    </tr>
-    <tr>
-        <td>Message</td>
-    </tr>
-    <tr>
-        <td><textarea cols="50" rows="5" name="message" wrap="off"> </textarea></td>
-    </tr>
-    <tr>
-        <td><br />
-        <input type="submit" value="Save" class="button"></td>
-    </tr>
+<div class="container">
+<form class="form-horizontal" role="form" name="apply_form" action="apply.html" method="post">
+<div class="form-group">
+    <div class="row">
+    <div class="col-md-4"><h2><b>Apply</b></h2></div>
+    <div class="col-md-4"></div>
+    <div class="col-md-4"></div>
+    </div><br />
+    <div class="row">
+    <div class="col-md-1">Username</div>
+    <div class="col-md-11">
+    <input type="text" name="username" class="form-control" placeholder="Username"></div>
+    </div><br />
+    <div class="row">
+    <div class="col-md-1">Job</div>
+    <div class="col-md-11">
+    <input type="text" name="occupation" class="form-control" placeholder="Occupation"></div>
+    </div><br />
+    <div class="row">
+    <div class="col-md-1">Message</div>
+    <div class="col-md-11">
+    <textarea cols="100" rows="5" class="form-control" name="message" wrap="off"> </textarea></div>
+    </div><br />
+    <div class="row">
+    <div class="col-md-2 col-md-offset-2">
+    <input type="submit" value="Save" class="btn btn-primary"></div>
```

Comment:
Không nên để `<div>` ở cuối dòng như này. Em xem format lại toàn bộ code của em nhé!